File: redcap_v15.7.6/API/examples/ruby/export_project_xml.rb
#!/usr/bin/env ruby

require 'curl'
require './settings.rb'
include Settings

fields = {
	:token => Settings::API_TOKEN,
    :content => 'project_xml',
    :returnMetadataOnly => 'false',
    :exportSurveyFields => 'false',
    :exportDataAccessGroups => 'false',
    :returnFormat => 'json'
}

ch = Curl::Easy.http_post(
  Settings::API_URL,
  fields.collect{|k, v| Curl::PostField.content(k.to_s, v)}
)
puts ch.body_str

File: redcap_v15.7.6/API/examples/r/export_project_xml.R
#!/usr/bin/env Rscript

source('config.R')
library(RCurl)

result <- postForm(
    api_url,
    token=api_token,
    content='project_xml',
    returnMetadataOnly='false',
    exportSurveyFields='false',
    exportDataAccessGroups='false',
    returnFormat='json'
)
print(result)
